internal/events/get.go

```diff
@@ -47,15 +64,26 @@ func get(
 	_ command.GlobalFlags,
 	services *services.Services,
 ) (command.Result, error) {
-	name := args[0]
-	start := args[1] // block height range start
-	end := ""        // block height range end
 
-	if len(args) == 3 {
-		end = args[2]
+	var start uint64
+	var end uint64
+	start = eventsFlags.Start
+	if eventsFlags.Start == 0 {
+		latestBlockHeight, err := services.Blocks.GetLatestBlockHeight()
+		if err != nil {
+			return nil, err
+		}
+		start = latestBlockHeight - eventsFlags.Last
+		end = latestBlockHeight
+	} else if eventsFlags.End == 0 {
+		latestBlockHeight, err := services.Blocks.GetLatestBlockHeight()
+		if err != nil {
+			return nil, err
+		}
+		end = latestBlockHeight
 	}
```

Comment:
Suggested fix (keep in mind this is not tested)

```suggestion
	start := eventsFlags.Start
	end := eventsFlags.End

	if start == 0 {
		end, err := services.Blocks.GetLatestBlockHeight()
		if err != nil {
			return nil, err
		}
		start = end - eventsFlags.Last
	} else if end == 0 {
		end, err := services.Blocks.GetLatestBlockHeight()
		if err != nil {
			return nil, err
		}
	}
```

The suggested fix is based on your logic, but at the same time should we have this logic? If start is not provided and end is by this logic we set the start and end to be latest and latest - last value. I think we should better handle arguments and provide a warning in case some are missing. Proposing (again not tested):

```suggestion
	start := eventsFlags.Start
	end := eventsFlags.End
	last := eventsFlags.Last

	// handle if not passing start and end
	if start == 0 && end == 0 {
		end, err := services.Blocks.GetLatestBlockHeight()
		if err != nil {
			return nil, err
		}
		
		start = end - last
	} else if start == 0 || end == 0 {
		return fmt.Errorf("please provide either both start and end for range or only last flag") 
	}
```